biosignals.py

```diff
@@ -0,0 +1,74 @@
+import platform
+import sys
+from dataclasses import dataclass, field
+
+import plux
+
+osDic = {"Darwin": "MacOS",
+         "Linux": "Linux64",
+         "Windows": ("Win32_37", "Win64_37")}
```

Comment:
I wasn't gonna add this, but you never know, I might have to work from a Mac, at some point.

Reply:
Always good to have full compatibility, especially when you start to work with teams or companies on your projects, or as you say - you may change your work machine.